imls-backend/db/migrations/20220811192329_heartbeat.sql

```diff
@@ -0,0 +1,27 @@
+-- migrate:up
+
+CREATE TABLE imlswifi.heartbeats (
+    heartbeat_id SERIAL PRIMARY KEY,
+    fscs_id VARCHAR(16) NOT NULL,
+    sensor_id SERIAL NOT NULL,
+    hourly_ping TIMESTAMP,
+    CONSTRAINT fk_heartbeat_library
+        FOREIGN KEY (fscs_id)
+            REFERENCES imlswifi.libraries(fscs_id),
+    CONSTRAINT fk_heartbeat_sensor
+        FOREIGN KEY (sensor_id)
+            REFERENCES imlswifi.sensors(sensor_id)
+
+);
+
+CREATE INDEX fk_heartbeat_library_index ON imlswifi.heartbeats(fscs_id);
+
+CREATE INDEX fk_heartbeat_sensor_index ON imlswifi.heartbeats(sensor_id);
+
+INSERT INTO imlswifi.heartbeats(sensor_id, fscs_id)
+    SELECT presences.sensor_id, presences.fscs_id
+     FROM imlswifi.presences;
```

Comment:
Why? I thought we were appending heartbeats. If we are writing over the heartbeat table repeatedly this makes sense. (Or, if DEFAULT is set as per above, that would give an automatic heartbeat stamp for all sensors).